docs/ops/arithmetic/Sinh_1.md

```diff
@@ -6,28 +6,28 @@
 
 **Short description**: *Sinh* performs element-wise hyperbolic sine (sinh) operation with given tensor.
 
-**Attributes**:
+**Detailed description**: *Sinh* does the following with the input tensor *a*:
```

Comment:
Let's keep the same style as for Cos spec.

\*\*Detailed description\*\*: \*Sinh\* performs element-wise hyperbolic sine (sinh) operation on a given input tensor, based on the following mathematical formula: